test/e2e/run_e2e.sh

```diff
@@ -0,0 +1,92 @@
+#!/bin/bash
+
+# Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+###############################################################################
+# PURPOSE: This script runs our pytest integration test suite.
```

Comment:
"e2e" not "integration"

Reply:
I see you found my copy/paste sections 😄